docs/gallery/interactive/pyfar_filtering.ipynb

```diff
@@ -0,0 +1,281 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Creating Filter objects\n",
+    "\n",
+    "pyfar stores filters in *Filter* ojects. These objects are used behind the scenes of the `pyfar.dsp.filter` module but can also be created by the user. Three different filter types are available in pyfar\n",
+    "\n",
+    "- **FIR Filter:** Finite Impulse Response (FIR) filters are non-recursive filters. FIR filters are very flexible and can have arbitrary magnitude and phase responses. One example for an FIR filter is an impulse response.\n",
+    "- **IIR Filter:** Infinite Impulse Response (IIR) filters are recursive filters. They can achieve steeper filter slopes than FIR filters of the same order but are less flexible with respect to the phase response.\n",
+    "- **SOS Filter:** Second Order Section (SOS) filters are cascaded 2nd order recursive filters. They are often more robust against numerical errors than IIR filters of the same order and are thus the most common way of realizing IIR filters.\n",
+    "\n",
+    "The application of a Filter is identical across the three types. Signals can be filtered using the `process` method of theFilter object. The filter types only differ in the way the filter coefficients are entered. The following examples show how to create and apply filter objects to audio signals using a simple IIR filter.\n",
+    "\n",
+    "A filter object is initialized with the filter coefficients and a sampling rate. For example, an IIR filter requires a set of b-coefficients and a-coefficients, like this simple comb filter example"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# design filter coefficients\n",
+    "b = [1, 0, 0]\n",
+    "a = [1, 0, .95]\n",
+    "coefficients = np.stack((b, a), 0)\n",
+    "\n",
+    "# create filter object\n",
+    "comb_filter = pf.FilterIIR(coefficients, 44100)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For convenience a summary of the filter can be obtained if it is printed"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(comb_filter)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Filtering a signal with a Filter object\n",
+    "\n",
+    "To filter an audio signal, pass it to the filter's process function. As a simple example, lets filter an impulse signal and plot the result to inspect the filter"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "input = pf.signals.impulse(256)\n",
+    "output = comb_filter.process(input)\n",
+    "\n",
+    "ax = pf.plot.time_freq(output, unit='samples')"
```

Comment:
```suggestion
    "input_signal = pf.signals.impulse(256)\n",
    "output_signal = comb_filter.process(input_signal)\n",
    "\n",
    "ax = pf.plot.time_freq(output_signal, unit='samples')"
```
To call it input is maybe not so nice, since its overwriting some internal function.

Reply:
True, changed it everywhere in the notebook